.github/workflows/create-wheels.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
   # four jobs are defined make-wheel-win-osx, make-wheel-linux, make-source-dist and make-emulated-wheels
   # the wheels jobs do the the same steps, but linux wheels need to be build to target manylinux
   make-wheel-win-osx:
+    needs: make-source-dist
```

Comment:
I've added a need, since in the source dist we verify that the version is correct etc